manifests/t/Teradata/TTUODBC/20.00.12.00/Teradata.TTUODBC.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Teradata.TTUOdbc
```

Comment:
Please update file name and path to match this line.

Reply:
Yes, Teradata.TTUOdbc is correct so we will change TTUODBC to TTUOdbc and resubmit. Thanks for the review.